modules/sso.php

```diff
@@ -1,4 +1,7 @@
 <?php
+
+use Automattic\Jetpack\Tracking\Manager as JetpackTracking;
```

Comment:
Nice, thanks for the fix! 

Several more files could use this though, maybe let's add it to them too:

```
/_inc/lib/admin-pages/class.jetpack-settings-page.php
/class.jetpack-client-server.php
/modules/plugin-search.php
/modules/stats.php
```

Reply:
Done in ac54d29d54d0c0cdccdec9151aaacaae9975acc8